src/scripts/html-transforms.js

```diff
@@ -1,24 +1,44 @@
+/**
+ * HtmlTransforms are functions that take an HTML Document and modify it in
+ * some useful way, such as removing scripts.
+ @typedef {(document: HTMLDocument) => HTMLDocument} HtmlTransform
+ */
+
+/**
+ * Takes several transforms and returns a new function that takes an html document,
+ * runs the transform functions on it, and returns the resulting document.
+ * @param {...HtmlTransform} The transforms to combine.
+ * @returns {HtmlTransform}
+ */
 export function compose (...transforms) {
   transforms = transforms.filter(transform => !!transform);
   if (transforms.length === 0) {
     return x => x;
   }
 
   return (input) => {
-    transforms.reduce((output, transform) => {
+    return transforms.reduce((output, transform) => {
       return transform(output);
     }, input);
   };
 }
 
+/**
+ * Takes an html document, removes all the stylesheets and scripts from
+ * the document. If any of them have a class or id that starts with 'wm-', 
+ * it keeps them as an exception. 
+ * Returns the resulting document.
+ * @param {HTMLDocument} The html document to change.
+ * @returns {HTMLDocument}
+ */
 export function removeStyleAndScript (document) {
   // Stylesheets and scripts
   document.querySelectorAll('link[rel="stylesheet"], style, script').forEach(node => {
     const isDiffNode = node.id.startsWith('wm-') ||
       Array.from(node.classList).some(name => name.startsWith('wm-'));
 
     if (!isDiffNode) {
-      node.remove();
```

Comment:
I hate to ask...but can you remove that extra whitespace at the end? My final comment on this PR :D